src/App.js

```diff
@@ -2,20 +2,92 @@ import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
 
-class App extends Component {
+class App extends Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      page: 0
+    };
+  }
+  onPageChange() {
+    this.setState( {
+      page: this.state.page + 1
+    } );
+  }
+
   render() {
+    if (this.state.page === 0) {
+      return <Title onDone = {this.onPageChange.bind(this)} />; 
+    }
+    else {
+      console.log("Hello World");
```

Comment:
This is great to use when developing!  When you submit pull requests, please remove `console.log` and other debugging code beforehand.